tests/test_redis.py

```diff
@@ -0,0 +1,30 @@
+import mock
+
+from health_check.contrib.redis.backends import RedisHealthCheck
+
+
+class TestRabbitMQHealthCheck:
+    """Test Redis health check."""
+
+    @mock.patch("health_check.contrib.redis.backends.getattr")
+    @mock.patch("health_check.contrib.redis.backends.Connection")
+    def test_redis_refused_connection(self, mocked_connection, mocked_getattr):
+        """Test when the connection to Redis is refused."""
+        mocked_getattr.return_value = "redis_url"
+
+        conn_exception = ConnectionRefusedError("Refused connection")
+
+        # mock returns
+        mocked_conn = mock.MagicMock()
```

Comment:
You test only one exception type here. Please test all of them.

Reply:
Fixed. 
I see `patch` and `project` still fails. I can't see `rabbitmq.apps` being tested, so I'm not sure if you want this to be tested too?